### PR TITLE
 Use podGroup instead of PDB for gang scheduling and upgrade k8s to 1.11.2

### PR DESCRIPTION
**Pull request by wackxu**
As title
/assign @gaocegege @suleisl2000

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/mxnet-operator/34)

### CODE REVIEW

File: pkg/controller.v1beta1/mxnet/controller.go [hunk 1]
```diff
@@ -72,6 +74,9 @@ type MXController struct {
 	// mxJobClientSet is a clientset for CRD MXJob.
 	mxJobClientSet mxjobclientset.Interface
 
+	//KubeBatchClientSet is a standard kube-batch clientset.
+	KubeBatchClientSet kubebatchclient.Interface
```
Comment (suleisl2000) [inline pkg/controller.v1beta1/mxnet/controller.go]: Why we add another member as KubeBatchClientSet here? I think it is already defined in JobController.

File: manifests/crd-podgroup.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```
Comment (suleisl2000) [inline manifests/crd-podgroup.yaml]: Suggest to change file name from crd-podgroup.yaml to podgroup.yaml.

Comment (wackxu) [reply manifests/crd-podgroup.yaml]: Done

**Comment by suleisl2000** [comment]
@wackxu Thanks for your contribution.

**Pull request merged**